### PR TITLE
Format README for publication

### PR DESCRIPTION
**Pull request by loafyloaf**
This adds a few sections to the README required for publication.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,49 @@
+[![Build Status](https://travis-ci.org/IBM/serverless-language-translation.svg?branch=master)](https://travis-ci.org/IBM/serverless-language-translation)
+
 # serverless-language-translation
```
Comment (stevemart) [inline README.md]: this should be an action or a complete sentence

File: README.md [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
 
-This repository contains a series of serverless functions that can serve as the backend for a multilingual chat room, as described in the blog post [here](https://medium.com/kkbankol-events/the-motivation-behind-this-particular-project-comes-from-playing-one-of-my-favorite-android-games-76c92b27c8e8)
+For this code pattern, this repository contains a series of serverless functions that can serve as the backend for a multilingual chat room.
```
Comment (stevemart) [inline README.md]: tihs sounds funny...

"This code pattern's repository contains... "

File: README.md [hunk 3]
```diff
@@ -1,13 +1,49 @@
+[![Build Status](https://travis-ci.org/IBM/serverless-language-translation.svg?branch=master)](https://travis-ci.org/IBM/serverless-language-translation)
+
 # serverless-language-translation
 
-This repository contains a series of serverless functions that can serve as the backend for a multilingual chat room, as described in the blog post [here](https://medium.com/kkbankol-events/the-motivation-behind-this-particular-project-comes-from-playing-one-of-my-favorite-android-games-76c92b27c8e8)
+For this code pattern, this repository contains a series of serverless functions that can serve as the backend for a multilingual chat room.  
+
+When the reader has completed this Code Pattern, they will understand how to:
+
+* deploy Cloud Function actions and triggers
```
Comment (stevemart) [inline README.md]: Capitalize!

File: README.md [hunk 4]
```diff
@@ -1,13 +1,49 @@
...
+* [Watson Text to Speech](https://www.ibm.com/watson/developercloud/text-to-speech.html): Converts written text into natural sounding audio in a variety of languages and voices.
+
+## Featured technologies
+
+* [Messaging](https://developer.ibm.com/messaging/message-hub/): Messaging is a key technology for modern applications using loosely decoupled architecture patterns such as microservices.
+* [Node.js](https://nodejs.org/): An open-source JavaScript run-time environment for executing server-side JavaScript code.
+* [Serverless](https://www.ibm.com/cloud-computing/bluemix/openwhisk): An event-action platform that allows you to execute code in response to an event.
+
+# Watch the Video
+![](https://imgur.com/Q4DGOPM.mp4)
```
Comment (stevemart) [inline README.md]: this isn't going to be the final video right?

File: README.md [hunk 5]
```diff
@@ -69,3 +102,9 @@ TODO:
 Create script to create wsk actions, populate credentials using .env files
 
 Port all issues from [Taiga kanban](https://tree.taiga.io/project/kalonb91-lang/) to this repository
+
```
Comment (stevemart) [inline README.md]: Should there be a section for "Running the application", like what to type in the chatroom?

Or a section on "Troubleshooting" to address common pitfalls?

File: README.md (hunk 4 above)
Comment (loafyloaf) [reply README.md]: that sounds more like a statement than a question... :)

**Pull request merged**